x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -126,6 +137,154 @@ export default function ({ getService }: FtrProviderContext) {
+          await ml.testExecution.logTestStep('checking page state before the cell selection');
+          await ml.anomalyExplorer.assertClearSelectionButtonVisible(false);
+          await ml.anomaliesTable.assertTableRowsCount(25);
+
+          await ml.testExecution.logTestStep('clicks on the Overall swim lane cell');
+          const sampleCell = (await ml.swimLane.getCells(overallSwimLaneTestSubj))[0];
+          await ml.swimLane.selectSingleCell(overallSwimLaneTestSubj, {
+            x: sampleCell.x + cellSize,
+            y: sampleCell.y + cellSize,
+          });
+          // TODO extend cell data with X and Y values, and cell width
+          await ml.swimLane.assertSelection(overallSwimLaneTestSubj, {
+            x: [1454846400000, 1454860800000],
+            y: ['Overall'],
+          });
+          await ml.anomalyExplorer.assertClearSelectionButtonVisible(true);
+
+          await ml.testExecution.logTestStep('updates the View By swim lane');
+          await ml.swimLane.assertAxisLabels(viewBySwimLaneTestSubj, 'y', ['EGF', 'DAL']);
+
+          await ml.testExecution.logTestStep('renders anomaly explorer charts');
+          await ml.anomalyExplorer.assertAnomalyExplorerChartsCount(4);
+
+          await ml.testExecution.logTestStep('updates top influencers list');
+          await ml.anomalyExplorer.assertInfluencerFieldListLength('airline', 2);
+
+          await ml.testExecution.logTestStep('updates anomalies table');
+          await ml.anomaliesTable.assertTableRowsCount(4);
+
+          await ml.testExecution.logTestStep('updates the URL state');
+          await ml.navigation.assertCurrentURL(
+            "/app/ml/explorer?_g=(ml%3A(jobIds%3A!(fq_multi_1_ae))%2CrefreshInterval%3A(display%3AOff%2Cpause%3A!t%2Cvalue%3A0)%2Ctime%3A(from%3A'2016-02-07T00%3A00%3A00.000Z'%2Cto%3A'2016-02-11T23%3A59%3A54.000Z'))&_a=(explorer%3A(mlExplorerFilter%3A()%2CmlExplorerSwimlane%3A(selectedLanes%3A!(Overall)%2CselectedTimes%3A!(1454846400%2C1454860800)%2CselectedType%3Aoverall%2CshowTopFieldValues%3A!t%2CviewByFieldName%3Aairline%2CviewByFromPage%3A1%2CviewByPerPage%3A10))%2Cquery%3A(query_string%3A(analyze_wildcard%3A!t%2Cquery%3A'*')))"
+          );
+
+          await ml.testExecution.logTestStep('clears the selection');
+          await ml.anomalyExplorer.clearSwimLaneSelection();
+          await ml.navigation.assertCurrentURL(
+            "/app/ml/explorer?_g=(ml%3A(jobIds%3A!(fq_multi_1_ae))%2CrefreshInterval%3A(display%3AOff%2Cpause%3A!t%2Cvalue%3A0)%2Ctime%3A(from%3A'2016-02-07T00%3A00%3A00.000Z'%2Cto%3A'2016-02-11T23%3A59%3A54.000Z'))&_a=(explorer%3A(mlExplorerFilter%3A()%2CmlExplorerSwimlane%3A(viewByFieldName%3Aairline%2CviewByFromPage%3A1%2CviewByPerPage%3A10))%2Cquery%3A(query_string%3A(analyze_wildcard%3A!t%2Cquery%3A'*')))"
+          );
+          await ml.anomalyExplorer.assertClearSelectionButtonVisible(false);
+          await ml.anomaliesTable.assertTableRowsCount(25);
```

Comment:
Might be good to also check that the influencers list is back to the original count and that the charts are not displayed anymore and that the axis labels are also back as expected.